src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -494,234 +395,85 @@ private void Register(Type t)
         /// <param name="serializerType">The type containing serialization methods for <paramref name="type"/>.</param>
         private void Register(Type type, Type serializerType)
         {
-            try
+            GetSerializationMethods(serializerType, out var copier, out var serializer, out var deserializer);
+
+            if (serializer == null && deserializer == null && copier == null)
```

Comment:
IS this a change in behavior? In the past, at least in this method we allowed to specify some of the delegates, but not necessarily all. (for example you could provide the copier, but not the serializer/deserializer pair, and the other way around). Then additional calls to this could enhance this and provide the missing delegate (or override just a few of them).
Not sure if that was a feature or just how it was implemented, but it's worth checking whether we are changing *expected* functionality

Reply:
That's still supported. These two guards throw if either:
* All three are null
* Serializer is null or Deserializer is null but not both